### ISSUE TITLE
The rest adapter docs is hard to follow

### ISSUE THREAD

**Issue by sltam** [emberjs/guides#349]
The rest adapter docs `http://guides.emberjs.com/v1.10.0/models/the-rest-adapter/` is using too many different pairs of names interchangeably, and this is simply confusing.

The actions table is explaining the endpoints for `store.find('post', 1)` so the table should show `/posts/123` instead of `/people/123`, etc.

Then the `inflector.irregular('formula', 'formulae');` example should be replaced by `inflector.irregular('person', 'people');` because this `person` vs `people` example is used again below.

**Comment by mixonic** [comment]
@sltam this comment is maybe better expressed as a PR. Can you whip one up?

**Comment by sltam** [comment]
I perfectly understand submitting a PR is the correct approach, but my employer does not allow me to contribute open source code unless I apply for approval, so submitting a PR is not a choice for me. Sorry.

**Comment by michaelrkn** [comment]
@sltam I totally agree that the use of different examples throughout all of the guides can make things a bit confusing. One of the goals for the guides in the next few months is to fix this up.

**Comment by locks** [comment]
@michaelrkn @sltam @mixonic I have addressed this issue in the PR referenced above. Notice though, that this change will only be deployed for the `1.13` guides (assuming it gets merged).

**Comment by michaelrkn** [comment]
@locks you are a hero

**Comment by sltam** [comment]
The fix looks perfect. Thanks!
